lib/matplotlib/quiver.py

```diff
@@ -562,18 +525,19 @@ def _init(self):
         """
         # It seems that there are not enough event notifications
         # available to have this work on an as-needed basis at present.
-        if True:  # not self._initialized:
+        if True:  # self._dpi_at_last_init != self.axes.figure.dpi
```

Comment:
Why did you change the comment?   why does this code get run unconditionally?

Reply:
I don't know why the original check got commented out (and don't really want to investigate right now), I'm just replacing _initialized by its newer moral equivalent.